### PR TITLE
Add | Crud for areas and search input

### PR DESCRIPTION
**Pull request by OchoaHub**
# Description
 - What?: I've added the CRUD for areas to implement QR Code, Institutions and areas. It includes 
model, table, controller and views.
 - Why?: These changes complete the areas creation experience.
 
 # Testing
Open terminal and type: 
1. yarn
2. bundle install
3. rails db:drop, rails db:create, rails db:migrate
4. rails s
5. And go to the link localhost/areas
6. That's it!
 
  ## Screenshots 
areaspr

### CODE REVIEW

File: app/controllers/areas_controller.rb [hunk 1]
```diff
@@ -23,23 +29,23 @@ def edit
   # POST /areas or /areas.json
   def create
     @area = Area.new(area_params)
-
+    @area.qrcode = generate_qrcode(@area.description)
     respond_to do |format|
       if @area.save
-        respond_if_is_true_web(@area, 'Symptom was successfully created.', :show, :created, @area)
+        respond_if_is_true_web(format, areas_url, 'Area was successfully created.', :show, :created, @area)
```
Comment (MiguelOcegueraM) [inline app/controllers/areas_controller.rb:35]: nice refactoring

**Review by MiguelOcegueraM — APPROVED** [review]
Man your team are on fire!! you are very fast bro

**Review by ArielMD — APPROVED** [review]
I liked how you added code QR, that looks good

**Review by fego-dev — APPROVED** [review]
The qrcode is so cool.
good job.

**Pull request merged**